### ISSUE TITLE
Try loading default config from /etc

### ISSUE THREAD

**Issue by jansol** [weechat/weechat#547]
In case no config was found in the home dir, weechat should try to load default values from /etc/weechat/*.conf
This would be useful for shell server maintainers: new (weechat) users would automatically get configs that adhere to server policies, maybe autojoin a relevant support channel etc...

**Comment by highflyer77** [comment]
You could easily write a wrapper script for weechat with bash, so that if no ~/.weechat/*.conf exists, then copy *.conf files from /etc/weechat/*.conf to the new ~/.weechat dir. Only then would it run "weechat". 
And/or you could also create the default weechat dir, upon creating a new user, using:  /etc/skel
Either way you would still need a variable or query for the nick of coarse.

**Comment by flashcode** [comment]
(reminder for myself: there was a task on savannah: https://savannah.nongnu.org/task/?5630)

**Comment by lp0** [comment]
This would also be useful for setting DCC configuration and triggers like #747, but it'd be better if the /etc file could be used to change the built-in defaults so that any changes would be applied on the next startup.

I really don't like ```/etc/skel``` because its content goes out of date very quickly and every user ends up with different behaviour. Similarly, copying the current default when weechat is first run means that changes never get applied.

**Comment by klloe** [comment]
@jansol Take a look at my [script](https://github.com/klloe/weechat-autoconf), you could use it as workaround to load the global config once when you setup the user.

**Comment by sudoforge** [comment]
I'd rather `/usr/lib/weechat` was used as the _base_ configuration directory. If I wanted to install `wee_slack.py` for all users, for example, I would expect that placing it in `/usr/lib/weechat/python/wee_slack.py` would allow it to be loaded by any user, and any user enabling autoloading would simply get a symlink in their `$HOME` back to the system loadpath.

TL;DR: Weechat should first source configuration and scripts from `/usr/lib/weechat`, and override with anything the user has set in `$HOME/.weechat` (or wherever their configuration is).

@flashcode, should this be created as a separate issue?

**Comment by sudoforge** [comment]
Just checking in to see if there are any new thoughts on this, @flashcode. I see the Savannah issue hasn't been updated either; is this something low priority enough that it will likely not make it onto the maintainers' board?

**Comment by flashcode** [comment]
@sudoforge: Savannah issues are not used at all any more (they're just closed if implemented with an issue like this one).
This feature has low priority and could be implemented in future, but we should discuss about what to implement. I think skeleton of config in /etc/weechat could be done, so on first startup, WeeChat will use config from there if defined, and if not create new config like now.
Using both /etc config and an own config after seems not easy to do for me.

**Comment by flashcode** [comment]
Anyway this would require a lot of changes, currently there's no static default config, it's automatically generated at runtime.
